src/WebJobs.Extensions.Http/WebJobs.Extensions.Http.csproj

```diff
@@ -1,12 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <Version>$(HttpVersion)</Version>
+    <Version>3.0.8.1</Version>
```

Comment:
@ConnorMcMahon I know you mentioned the need to update the assembly version but not sure if this is what you meant.

Reply:
Ah, you actually don't need to update the assembly version. You just need to update the nuget version everytime you test locally to avoid hitting the cached nuget version.

In general, you will want to update the Version before this releases, but I think you will want to edit the $(HttpVersion) variable. I think that comes from the common.props file?